backend/workflow_manager/endpoint_v2/source.py

```diff
@@ -256,8 +250,7 @@ def publish_input_file_content(self, input_file_path: str, input_text: str) -> N
             return None
         output_log_message = f"## Input text:\n\n```text\n{input_text}\n```\n\n"
         input_log_message = (
-            "## Input file:\n\n```text\n"
-            f"{os.path.basename(input_file_path)}\n```\n\n"
+            "## Input file:\n\n```text\n" f"{os.path.basename(input_file_path)}\n```\n\n"
```

Comment:
```suggestion
            f"## Input file:\n\n```text\n{os.path.basename(input_file_path)}\n```\n\n"
```